src/battle.c

```diff
@@ -2048,6 +2048,25 @@ void dazzle(battle * b, troop * td)
     td->fighter->person[td->index].defence--;
 }
 
+int new_castle_rule(void)
```

Comment:
Könnte man das bitte in was Aussagekräftiges umbenennen. new_ ist praktisch nie ein guter Name für irgendwas.